### PR TITLE
Basic podcasts interaction

### PR DESCRIPTION
**Pull request by oilnam**

### CODE REVIEW

File: src/speech.js [hunk 1]
```diff
@@ -80,11 +80,17 @@ const speech = {
...
-        "Absolutely, ",
-        "No problem, "
-    ]
+        'Sure, ',
+        'Certainly, ',
+        'Absolutely, ',
+        'No problem, '
+    ],
+    podcasts: {
+        intro: 'Our most popular podcasts are: Football Weekly, Politics Weekly, Close Encounters and Science Weekly. ' +
```
Comment (chriswilk) [inline src/speech.js]: @oilnam this is best I've come up with so far:
```
Our most popular podcasts this week are: Football Weekly, Science Weekly, Chips with Everything, and Close Encounters. I’ve sent you the list of podcasts to your Alexa app. I can tell you the latest available or play you a specific one. What would you like to hear?
```

File: src/helpers.js [hunk 2]
```diff
@@ -56,3 +56,44 @@ exports.getTopic = (attributes, slots) => {
         default: return (slots.topic && slots.topic.value) ? slots.topic.value : null;
     }
 };
+
+exports.getPodcastDirective = (podcastUrl) => {
```
Comment (tomrf1) [inline src/helpers.js]: just a style preference thing, but I wouldn't have defined the object outside of this function, there's no need

File: src/intentLogic/getPodcast.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+const get = require('simple-get-promise').get;
+const asJson = require('simple-get-promise').asJson;
+//const Promise = require('promise')
```
Comment (tomrf1) [inline src/intentLogic/getPodcast.js]: my fault, this is from when I was playing with promises :)

File: src/intentLogic/yes.js [hunk 4]
```diff
@@ -6,15 +6,15 @@ const speech = require('../speech').speech;
...
-    } else {
-        this.emit(':tell', speech.core.stop)
+    const attributes = this.event.session.attributes;
+
+    switch (attributes.lastIntent) {
+        case 'GetPodcastIntent':
+            const podcastDirective = helpers.getPodcastDirective(attributes.podcastUrl);
+            this.emit('PlayPodcastIntent', podcastDirective);
+            break;
+        default: return null;
```
Comment (tomrf1) [inline src/intentLogic/yes.js]: this crashes the skill, can we do something like `this.emit(':ask', speech.help.reprompt)` instead?

**Comment by tomrf1** [comment]
👍 
(squash)

**Pull request merged**